minio/minio
# Helm chart: Add support for custom bucket policies

## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license] (https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
Adds a way to set custom policies via `mc anonymous set-json` for buckets created via the Helm chart.

## Motivation and Context
We have a lot of custom policies set via the UI on buckets and If we try to use the Helm chart to create these buckets it will always override the set policies.

## How to test this PR?
Uncomment example `bucket3` in `values.yaml` and verify that a custom policy is set using `mc anonymous get-json myminio/bucket3`.

## Types of changes

## Checklist:
